src/Http/Body/Encoder/MultipartFormDataEncoder.php

```diff
@@ -24,18 +24,11 @@
  */
 class MultipartFormDataEncoder implements Encoder
 {
-    /** @var string */
-    private $boundary;
-
-    /** @var AppendableStream */
-    private $multiPartStream;
-
-    /** @var string */
-    private $crlf = "\r\n";
+    private string $boundary;
+    private AppendableStream $multiPartStream;
+    private string $crlf = "\r\n";
```

Comment:
ist out of scope des Intents dieses PRs, aber warum ist das variabel? wenn sich die Gesetze des Universums ändern und CR LF nicht mehr "\r\n" ist, darf unser Programm ruhig nicht mehr kompilieren 🤨